english/fruit/devilfruits/df.md

```diff
@@ -0,0 +1,3 @@
+A Devil Fruit (悪魔の実 Akuma no Mi) is a type of fruit that, which when eaten, confers a power on the consumer. There are three categories of Devil Fruits; Zoan (動物系ゾオン Zoon) fruits allow the user to fully and partially transform into a specific animal; as well as real-life animals. Some Zoan fruits allow the user to transform into mythical creatures. Logia (自然系ロギア Rogia) fruits give control over and allow the user "to change their living body structure into the powers of nature". Paramecia (超人系パラミシア Paramishia) is a category for fruits that give the user superhuman abilities. They are said to be incarnations of the sea devil himself, and as a result, Devil Fruit users cannot swim in sea water, as "they are hated by the sea".
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.